src/components/AmountEditable.tsx

```diff
@@ -306,27 +400,57 @@ export const AmountEditable: ParentComponent<{
             character === "DEL" ||
             character === i18n.t("receive.amount_editable.del")
         ) {
-            if (localValue().length <= 1) {
+            if (
```

Comment:
another case where there's enough logic here that some examples in the comments would be helpful